brew/src/test/java/nl/tudelft/serg/evosql/brew/DataGenerator.java

```diff
@@ -1,6 +1,7 @@
 package nl.tudelft.serg.evosql.brew;
 
 import nl.tudelft.serg.evosql.brew.data.*;
+import nl.tudelft.serg.evosql.metaheuristics.DatabaseOutput;
```

Comment:
This import creates an unnecessary dependency on `ga`. I would suggest simply using `List<HashMap<String, String>>` as you did elsewhere.

Reply:
Sure